_extras/guide.md

```diff
@@ -247,6 +247,28 @@ group is interested.
 
 ### Afternoon Wrap-Up
 
+## VI. Teaching Demonstration Tips
+
+Suggested points to make during introductions:
+* This is not a high-stakes test! This is a friendly opportunity to give and receive feedback. On occasion, we do invite 
+people to return for a second try. If we do this, your follow-up email will contain very clear instructions on exactly what
+we'd like you to change on your next visit. This does not mean we don't want you as an instructor, it means we want you to be
+ready to put your best foot forward in the classroom and we are here to help you do that. (KW)
+* I will not tell anyone that they have passed or ask anyone to repeat during this session. That information will come in the follow up email. (KW)
+* The feedback you give sometimes says as much or more about your approach to instruction as the demo itself. (KW)
+
+Different ways of managing feedback during the demo:
+* Create your own etherpad and populate it with episode links, feedback templates, whatever. 
+* Use +/- content delivery rubric. (Or use "improvement" in place of "-" as suggested by MC). Keep in mind that some learners
```

Comment:
Do you mean the '2x2' grid we use during training? Asking for feedback on delivery makes much sense, but I always feel asking for feedback on content feels like asking for feedback on the lesson material, which is beyond the control of the trainee...